v2/concordium/service.proto

```diff
@@ -277,4 +277,29 @@ service Queries {
   // If the endpoint is not enabled by the node, then an 'unimplemented' error
   // will be returned.
   rpc GetBlockCertificates(BlockHashInput) returns (BlockCertificates);
+
+  // Get the list of bakers that won the lottery in a particular historical epoch (i.e. the
+  // last finalized block is in a later epoch). This lists the winners for each round in the
+  // epoch, starting from the round after the last block in the previous epoch, running to
+  // the round before the first block in the next epoch. It also indicates if a block in each
+  // round was included in the finalized chain.
+  //
+  // The following error cases are possible:
+  //  * `NOT_FOUND` if the query specifies an unknown block.
+  //  * `UNAVAILABLE` if the query is for an epoch that is not finalized in the current genesis
+  ///    index, or is for a future genesis index.
+  //  * `INVALID_ARGUMENT` if the query is for an epoch that is not finalized for a past genesis
+  //    index.
```

Comment:
BlockHashInput can also be malformed, so in that case it can return INVALID_ARGUMENT as well.

In principle this can also return UNIMPLEMENTED if the endpoint is disabled.